api/conf/settings/rest_framework.py

```diff
@@ -13,6 +13,45 @@
 THROTTLE_OAUTH2_THUMBS = config("THROTTLE_OAUTH2_THUMBS", default="500/minute")
 THROTTLE_ANON_HEALTHCHECK = config("THROTTLE_ANON_HEALTHCHECK", default="3/minute")
 
+THROTTLE_OV_REFERRER_BURST = config(
+    "THROTTLE_OV_REFERRER_BURST", default=THROTTLE_ANON_BURST
+)
+THROTTLE_OV_REFERRER_SUSTAINED = config(
+    "THROTTLE_OV_REFERRER_SUSTAINED", default=THROTTLE_ANON_SUSTAINED
+)
+THROTTLE_OV_REFERRER_THUMBS = config(
+    "THROTTLE_OV_REFERRER_THUMBS", default=THROTTLE_ANON_THUMBS
+)
+
+DEFAULT_THROTTLE_RATES = {
+    "anon_burst": THROTTLE_ANON_BURST,
+    "anon_sustained": THROTTLE_ANON_SUSTAINED,
+    "anon_healthcheck": THROTTLE_ANON_HEALTHCHECK,
+    "anon_thumbnail": THROTTLE_ANON_THUMBS,
+    "ov_referrer_burst": THROTTLE_OV_REFERRER_BURST,
+    "ov_referrer_sustained": THROTTLE_OV_REFERRER_SUSTAINED,
+    "ov_referrer_thumbnail": THROTTLE_OV_REFERRER_THUMBS,
+    "oauth2_client_credentials_thumbnail": THROTTLE_OAUTH2_THUMBS,
+    "oauth2_client_credentials_sustained": "10000/day",
+    "oauth2_client_credentials_burst": "100/min",
+    "enhanced_oauth2_client_credentials_sustained": "20000/day",
+    "enhanced_oauth2_client_credentials_burst": "200/min",
+    # ``None`` completely by-passes the rate limiting
+    "exempt_oauth2_client_credentials": None,
+}
+
+DEFAULT_THROTTLE_CLASSES = (
+    "api.utils.throttle.BurstRateThrottle",
+    "api.utils.throttle.SustainedRateThrottle",
+    "api.utils.throttle.OpenverseReferrerBurstRateThrottle",
+    "api.utils.throttle.OpenverseReferrerSustainedRateThrottle",
+    "api.utils.throttle.OAuth2IdBurstRateThrottle",
+    "api.utils.throttle.OAuth2IdSustainedRateThrottle",
+    "api.utils.throttle.EnhancedOAuth2IdBurstRateThrottle",
+    "api.utils.throttle.EnhancedOAuth2IdSustainedRateThrottle",
+    "api.utils.throttle.ExemptOAuth2IdRateThrottle",
+)
```

Comment:
I moved these so that they are easier to reference in the tests when we need to enable the throttling for a scoped set of tests.